docs/howto/csrf.txt

```diff
@@ -247,8 +247,8 @@ Solution: rather than disabling the middleware and applying ``csrf_protect`` to
 all the views that need it, enable the middleware and use
 :func:`~django.views.decorators.csrf.csrf_exempt`.
 
-Setting the token when CsrfViewMiddleware.process_view is not used
-------------------------------------------------------------------
+Setting the token when ``CsrfViewMiddleware.process_view()`` is not used
+------------------------------------------------------------------------
```

Comment:
Can you confirm you are happy this is a `---` level heading and not a `===`.

Reply:
Yes, it's in `Edge cases` as before.